frontend/src/UserAPI.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const getRoot = `https://jsonplaceholder.typicode.com/users`;
+
+function GetUsers() {
+    return axios
```

Comment:
Lehet axios instance-t csinálni, így a domain-t nem kell majd megadni minden kérésnél, csak az instance-ben, valami ilyesmi módon:
```
// file elején
const api = axios.create({
  baseURL: 'https://some-domain.com/api/',
}
...
// metódusban
return api.get('/users')
```

Ez akkor lesz hasznos, majd ha több végpont is lesz ;)